NFIQ2/NFIQ2Algorithm/include/nfiq2_data.hpp

```diff
@@ -8,91 +8,85 @@
 #include <vector>
 
 namespace NFIQ2 {
+
 /**
-******************************************************************************
-* @class Data
-* @brief This class manages binary data and offers functions for easy handling
-* of data. Examples of binary data are proprietary fingerprint templates or
-* quality feature values in binary format.
-******************************************************************************/
+ * 	This class manages binary data and offers functions for easy handling
+ * 	of data. Examples of binary data are proprietary fingerprint templates
+ * 	or quality feature values in binary format.
+ */
 class Data : public std::basic_string<uint8_t> {
     public:
-	/******************************************************************************/
-	// --- Constructor / Destructor --- //
-	/******************************************************************************/
-
-	/**
-	 * @brief Default constructor of Data
-	 */
+	/** Default Data Constructor. */
 	Data();
 
-	/**
-	 * @brief Constructor of data with available pointer to data
-	 */
+	/** Constructor with available pointer to data. */
 	Data(const uint8_t *pData, uint32_t dataSize);
 
-	/**
-	 * @brief Copy constructor
-	 */
+	/** Copy constructor. */
 	Data(const Data &otherData);
 
-	/**
-	 * @brief Copy constructor
-	 */
+	/** Copy constructor. */
 	explicit Data(const std::basic_string<uint8_t> &otherData);
 
-	/**
-	 * @brief Destructor
-	 */
+	/** Destructor. */
 	virtual ~Data();
 
-	/******************************************************************************/
-	// --- Functions --- //
-	/******************************************************************************/
-
 	/**
-	 * @fn readFromfile
-	 * @brief Reads the content from the specified filename into the data
+	 * @brief
+	 * Reads the content from the specified `filename` into the data
```

Comment:
Really brief shouldn't use the params. It's used for folks trying to learn what a function does.

```suggestion
	 * Reads the content from the a file into the data
```